dashboard/src/components/AppView/AppView.tsx

```diff
@@ -39,6 +39,7 @@ interface IAppViewState {
   services: Array<IKubeItem<IResource>>;
   serviceRefs: ResourceRef[];
   ingresses: Array<IKubeItem<IResource>>;
```

Comment:
Do we need ingresses anymore? And services?

Reply:
Yep, we can remove these now.